can_messages.asciipb

```diff
@@ -0,0 +1,263 @@
+msg {
+  id:0
+  msg_name: "bps fault"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:1
+  msg_name: "motor drive"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:2
+  msg_name: "motor cruise"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:3
+  msg_name: "change turn signal state"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:4
+  msg_name: "hazard"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:5
+  msg_name: "headlight state"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:6
+  msg_name: "brake light state"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:7
+  msg_name: "horn state"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:8
+  msg_name: "set bus current limit"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:9
+  msg_name: "heartbeat request"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:10
+  msg_name: "set power state"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:11
+  msg_name: "throttle position"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:12
+  msg_name: "mechanical brake position"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:13
+  msg_name: "battery module voltage"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:14
+  msg_name: "battery current"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:15
+  msg_name: "battery module temperature"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:16
+  msg_name: "battery state of charge"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+
+msg {
+  id:17
+  msg_name: "solar voltage"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:18
+  msg_name: "solar power"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:19
+  msg_name: "solar current"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:20
+  msg_name: "motor speed"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:21
+  msg_name: "motor current"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:22
+  msg_name: "motor voltage"
+  can_data {
+    frame {
+      type: UINT64
+    }
+  }
+}
+
+msg {
+  id:23
+  msg_name: "heartbeat response"
+  msg_type: true
```

Comment:
We can simply rename this field to better reflect its purpose&mdash;``is_ack`` or something.